theme/less/homepage.less

```diff
@@ -62,8 +62,10 @@ main article > *:first-child
   margin: 0!important;
 }
 
-.box h3 
+.box h2 
 {
+  font-size: 1.4em;
+  line-height: 120%;
```

Comment:
Note: I did notice that we are using weird `font-size` and `line-height` sequence, where line-height is using smaller em value, but produces higher result, probably because higher line-height is set on parent. Thinking switching to percentage, make more sense to avoid such issues. Wonder if we should address that everywhere else?

Reply:
Created issue for that -> https://github.com/browser-automation/cba-website/issues/44